src/server/middleware/missingBundleMiddleware.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ */
+const dedent = require('dedent');
+
+function missingBundleMiddleware(req, res, next) {
+  // eslint-disable-next-line no-unused-vars
+  const [_, platform] = req.path.match(/(?:\.(ios|android))?\.bundle/) || [];
+
+  if (platform) {
+    res.status(500).end(
+      dedent`
+      Bundle couldn't be loaded.
+
+      Try the following to fix the issue:
+      - Ensure that Haul is running with --platform all or --platform ${platform}
+      - Ensure that bundle has compiled successfuly by checking terminal window where Haul is running
```

Comment:
Will this ever happen? I thought webpack doesn't respond to the request until bundle is finished